Dockerfile

```diff
@@ -62,6 +62,7 @@ env CGO_ENABLED=0 \
 COPY . /go/src/github.com/vmware-tanzu/velero
 
 RUN mkdir -p /output/usr/bin && \
+    export GOARM=$( echo "${GOARM}" | cut -c2-) && \
```

Comment:
IMO it'd be slightly better: `bash GOARM=$( echo "${GOARM}" | cut -c2-) /go/src/github.com/vmware-tanzu/velero/hack/build-restic.sh

Reply:
Cannot use `Key=Value` format here. The variable and command in it are not evaluated.
Removed the bash before `build-restic.sh`